src/chunk.c

```diff
@@ -2266,6 +2266,15 @@ ts_chunk_get_by_relid(Oid relid, bool fail_if_not_found)
 
 	schema = get_namespace_name(get_rel_namespace(relid));
```

Comment:
hmm if get_rel_namespace fails everything below will fail too cause there is no matching entry in pg_class

Reply:
It could return a valid schema, that's why the check for either schema or relname being NULL